tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup.yaml

```diff
@@ -18,7 +18,7 @@ spec:
           katacontainers.io/kata-runtime: cleanup
       containers:
       - name: kube-kata-cleanup
-        image: katadocker/kata-deploy
+        image: katadocker/kata-deploy:2.0.2
```

Comment:
Why 2.0.2 and not 2.0.3?

Reply:
@c3d, firstly, thanks for the review.

> Looks OK except for a stray 2.0.2 (not sure if this is intentional)
This is intentional because `./tools/packaging/release/update-repository-version.sh` will update from the current version (2.0.2) to the new version (2.0.3) automatically, and open a PR with the change.

Right now, as we didn't have any tag there (and, consequently, people were using `:latest`, which is 1.x) the easy way to have the script fully working without adding crazy logic to it was adding the **current** version tag to the image.

Makes sense?